contracts/TreasuryManager.sol

```diff
@@ -28,24 +25,25 @@ contract TreasuryManager is
     /// @notice precision used to limit the amount of NOTE price impact (1e8 = 100%)
     uint256 internal constant NOTE_PURCHASE_LIMIT_PRECISION = 1e8;
 
+    // following constants are relevant only for mainnet
+    IERC20 public constant NOTE = IERC20(0xCFEAead4947f0705A14ec42aC3D44129E1Ef3eD5);
+    IStakedNote public constant sNOTE = IStakedNote(0x38DE42F4BA8a35056b33A746A6b45bE9B1c3B9d2);
+    bytes32 public constant NOTE_ETH_POOL_ID = 0x5122e01d819e58bb2e22528c0d68d310f0aa6fd7000200000000000000000163;
+    ERC20 public constant BALANCER_POOL_TOKEN = ERC20(0x5122E01D819E58BB2E22528c0D68D310f0AA6FD7);
+    IVault public constant BALANCER_VAULT = IVault(0xBA12222222228d8Ba445958a75a0704d566BF2C8);
+    uint256 public constant WETH_INDEX = 0;
+    uint256 public constant NOTE_INDEX = 1;
+
     NotionalTreasuryAction public immutable NOTIONAL;
-    IERC20 public immutable NOTE;
-    IVault public immutable BALANCER_VAULT;
-    ERC20 public immutable BALANCER_POOL_TOKEN;
-    IStakedNote public immutable sNOTE;
-    bytes32 public immutable NOTE_ETH_POOL_ID;
-    address public immutable ASSET_PROXY;
-    IExchangeV3 public immutable EXCHANGE;
+    WETH9 public immutable WETH;
     ITradingModule public immutable TRADING_MODULE;
     uint32 public constant MAXIMUM_COOL_DOWN_PERIOD_SECONDS = 30 days;
-    
+
     /// @notice From IPriceOracle.getLargestSafeQueryWindow
     uint32 public constant MAX_ORACLE_WINDOW_SIZE = 122400;
 
     /// @notice Balancer token indexes
     /// Balancer requires token addresses to be sorted BAL#102
-    uint256 public immutable WETH_INDEX;
-    uint256 public immutable NOTE_INDEX;
 
     address public manager;
 
```

Comment:
Can you add a test where you run this upgrade on both mainnet and arbitrum and ensure that all the storage variables do not change before and after the upgrade?